.travis.yml

```diff
@@ -6,9 +6,16 @@ env:
     - CI_TARGET=vim MAKTABA_VERSION=1.14.0
     - CI_TARGET=vim MAKTABA_VERSION=master
     - CI_TARGET=neovim MAKTABA_VERSION=master
+    - CI_TARGET=vim MAKTABA_VERSION=master COVERAGEPY="coverage>4,<5"
 before_script:
   - sudo apt-get update
-  - sudo apt-get install python3-dev python3-coverage
+  - sudo apt-get install python3-dev
+  - if [ -z "$COVERAGEPY" ]; then
+      sudo apt-get install python-coverage3;
+    else
+      wget https://bootstrap.pypa.io/get-pip.py &&
```

Comment:
Don't you need to execute this after downloading it? Otherwise I think you're running the system version of pip below.

Here's a working version from one of our other .travis.yml files: https://github.com/google/vim-codefmt/blob/c84463dbed1408abb4003b2105211488d47423d9/.travis.yml#L18
That fetches for version 3.5 explicitly or it blows up on Travis not running python >=3.6, and we sudo to install the packages "system-wide" on the container.

Reply:
I've removed the wget - it seems using the system Python is fine here.

Have not looked at the other reference - I am not sure if putting more time into the Travis config is worth it, given that it likely will run out of OSS credits in the long run anyway, and needs to be replaced by e.g. GitHub Actions then.